kore/src/Kore/Builtin/String.hs

```diff
@@ -335,6 +341,19 @@ evalString2Int = Builtin.functionEvaluator evalString2Int0
             _ -> return (Pattern.bottomOf resultSort)
     evalString2Int0 _ _ _ = Builtin.wrongArity string2IntKey
 
+evalBase2String :: BuiltinAndAxiomSimplifier
+evalBase2String = Builtin.functionEvaluator evalBase2String0
+  where
+    evalBase2String0 _ resultSort [_int, _base] = do
+      _int <- Int.expectBuiltinInt base2StringKey _int
+      _base <- Int.expectBuiltinInt base2StringKey _base
+      return $ if 2 <= _base && _base <= 36
+        then let charRep = ("0123456789abcdefghijklmnopqrstuvwxyz" !!)
+                 showWithBase = showSigned (showIntAtBase _base charRep) 0
+             in asPattern resultSort $ Text.pack $ showWithBase _int ""
+        else Pattern.bottomOf resultSort
```

Comment:
`\bottom` means that the input is invalid. In this case, the input is valid, but the built-in function simply isn't implemented. Use the `warnNotImplemented` code from above.

```suggestion
        else warnNotImplemented app >> empty
```